README.md

```diff
@@ -1,5 +1,39 @@
-# talkable-ios-sdk
-Talkable Integration and API for iOS devices.
+# Talkable iOS SDK
+
+[![Version](https://img.shields.io/cocoapods/v/TalkableSDK.svg?style=flat-square)](http://cocoapods.org/pods/TalkableSDK)
+[![License](https://img.shields.io/cocoapods/l/TalkableSDK.svg?style=flat-square)](http://cocoapods.org/pods/TalkableSDK)
+[![Platform](https://img.shields.io/cocoapods/p/TalkableSDK.svg?style=flat-square)](http://cocoapods.org/pods/TalkableSDK)
+
+Talkable SDK makes it easy to integrate Talkable referral functionality into iOS apps.
+
+## Installation
+
+## Requirements
+- [x] Xcode 11.
```

Comment:
Do we actually require Xcode 11?

Reply:
We can remove this line, because i tested our framework with Xcode 10.3. And it's work. But it will be better to use Xcode 11. Because, we compile with Xcode 11 Command Line Tools.

Reply:
So lets remove it

Reply:
Done